src/assets/main.scss

```diff
@@ -16,6 +16,45 @@ body {
   text-rendering: optimizeLegibility;
   -webkit-font-smoothing: antialiased;
   -moz-osx-font-smoothing: grayscale;
+  margin-bottom: 50px;
+}
+
+a,
+a:link,
+a:visited,
+a:hover,
+a:active {
+  text-decoration: none;
+  color: inherit;
+}
+
+li {
+  list-style-type: none;
+  padding: 0;
+  margin: 0;
+}
+
+// OVERWRITES
+
+#app {
```

Comment:
Maybe there is no need to override it here? Especially if it's high specificity styling it's usually better to keep those styles in scoped sheets so that you don't have suprises later on